backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/process/JavaProcess.java

```diff
@@ -752,7 +752,7 @@ public final Properties getCtx()
 		final Properties ctx = _ctx;
 		if (ctx == null)
 		{
-			log.warn("No context configured for {}. Returning global context", this);
+			log.debug("No context configured for {}. Returning global context", this);
```

Comment:
why? what's the case when ctx is null?

Reply:
For example if menu with processes for doc_outbound_log is opened, this `2024-12-11 17:24:28.723  WARN 12 --- [https-jsse-nio-8443-exec-10] d.m.edi.process.EDIExportDocOutboundLog  : No context configured for EDIExportDocOutboundLog{processInfo=null}. Returning global context` is logged as it seems for each record. So this results in thousends of logged lines

Reply:
Another example is `2024-12-11 06:31:26.332  WARN 166 --- [https-jsse-nio-8443-exec-6] e_Candidate_EnqueueSelectionForInvoicing : No context configured for C_Invoice_Candidate_EnqueueSelectionForInvoicing{processInfo=null}. Returning global context` but there only once